interfaces/mount/entry.go

```diff
@@ -103,3 +119,28 @@ func (e Entry) String() string {
 	return fmt.Sprintf("%s %s %s %s %d %d",
 		name, dir, fsType, options, e.DumpFrequency, e.CheckPassNumber)
 }
+
+// ParseEntry parses a fstab-like entry.
+func ParseEntry(s string) (Entry, error) {
+	var e Entry
+	fields := strings.Fields(s)
+	// do all error checks before any assignments to `e'
+	if len(fields) != 6 {
```

Comment:
Aha, interesting.

The documented format of fstab doesn't have the extra entries and I don't think we should handle those. The optional aspect of the last two fields is indeed something that is easy to handle so I'll do that quickly. Thanks for noticing that!